airbyte-integrations/connectors/destination-snowflake/src/main/java/io/airbyte/integrations/destination/snowflake/SnowflakeSqlOperations.java

```diff
@@ -106,6 +106,11 @@ public void insertRecordsInternal(final JdbcDatabase database,
     SqlOperationsUtils.insertRawRecordsInSingleQuery(insertQuery, recordQuery, database, records);
   }
 
+  @Override
+  protected void insertRecordsInternalV2(final JdbcDatabase jdbcDatabase, final List<PartialAirbyteMessage> list, final String s, final String s1) throws Exception {
```

Comment:
I'm not sure we ever want to do this? Since snowflake only has an internal staging loading method, i.e. standard inserts doesn't exist at all

Reply:
So the insertRecordsInternal is also dead code path ?

Reply:
wasn't sure, did some digging just now. I suspect it's used as part of the `check` method... which it probably _shouldn't_ be. Specifically [this](https://github.com/airbytehq/airbyte/blob/8c592fec4dc63c7978ae09d98ff24a452b058414/airbyte-cdk/java/airbyte-cdk/db-destinations/src/main/java/io/airbyte/cdk/integrations/destination/jdbc/AbstractJdbcDestination.java#L160) line dispatches to JdbcSqlOperations, which [invokes](https://github.com/airbytehq/airbyte/blob/8c592fec4dc63c7978ae09d98ff24a452b058414/airbyte-cdk/java/airbyte-cdk/db-destinations/src/main/java/io/airbyte/cdk/integrations/destination/jdbc/JdbcSqlOperations.java#L191) `insertRecordsInternal`.

but otherwise yes, dead code